internal/pkg/service/appsproxy/http/router.go

```diff
@@ -715,3 +721,58 @@ func headerFromClaim(header, claim string) options.Header {
 		},
 	}
 }
+
+func (r *Router) getHTMLPage(ctx context.Context, w http.ResponseWriter, imageFilePath string, htmlFilePath string, exceptionID string, appID string) {
+	// Encode the GIF data to Base64.
+	imageBase64, err := encodeImageDataToBase64(imageFilePath)
```

Comment:
The `Inline` base64 to html is too long, so we will parse it in the code and then insert it into the template as a placeholder